src/animalShelter/AnimalShelter.ts

```diff
@@ -6,20 +6,18 @@ import Squirrel from "../animals/Squirrel";
 import Tortoise from "../animals/Tortoise";
 import Snake from "../animals/Snake";
 import Animal from "../hierarchy/Animal";
-const DEFAULT_MAMMAL_LIMIT = 20;
-const DEFAULT_REPTILE_LIMIT = 19;
+import Reptile from "../hierarchy/Reptile";
+const ANIMAL_SHELTER_MAMMAL_LIMIT = 20;
+const ANIMAL_SHELTER_REPTILE_LIMIT = 19;
 export default class AnimalShelter {
   private static instance: AnimalShelter;
-  private _mammalLimit: number = DEFAULT_MAMMAL_LIMIT;
-  private _reptileLimit: number = DEFAULT_REPTILE_LIMIT;
-  private lions = 0;
-  private crocodiles = 0;
-  private snakes = 0;
-  private squirrels = 0;
-  private tortoise = 0;
+  mammalLimit: number = ANIMAL_SHELTER_MAMMAL_LIMIT;
+  reptileLimit: number = ANIMAL_SHELTER_REPTILE_LIMIT;
+
   private animals: Animal[] = [];
 
-  private constructor() {}
+  private constructor() { }
+
```

Comment:
before commiting you can go to Visual Studio's source control and see all changes you are about to commit. If you see any unnecessary ones like this one you should discard them so they don't end up in the PR